app/react/portainer/environments/wizard/components/Option/Option.tsx

```diff
@@ -44,11 +45,11 @@ export function Option({
       disabled={isLimited}
       onClick={onClick}
     >
-      <div className="text-center mt-2">
-        {Icon ? (
-          <Icon selected={active} className={styles.iconComponent} />
+      <div className="flex items-end justify-center text-center mt-2">
+        {IconComponent ? (
+          <IconComponent selected={active} className={styles.iconComponent} />
         ) : (
-          <i className={clsx(icon, 'block', styles.icon)} />
+          <Icon icon={icon} className={clsx('block', styles.icon)} />
         )}
```

Comment:
this snippet was needed because we had `KaaSIcon` and string icons. now all of them are components, so we can just do:
```ts
const IconComponent = icon;

<div className="flex items-end justify-center text-center mt-2">
    <IconComponent selected={active} className={styles.iconComponent} />
</div>
```

Reply:
updated